packages/terra-abstract-modal/src/AbstractModal.scss

```diff
@@ -0,0 +1,26 @@
+:local {
+  .abstract-modal {
+    display: block;
+    left: 50%;
+    max-height: calc(100% - 2 * var(--terra-abstract-modal-vertical-inset, 10px));
+    max-width: calc(100% - 2 * var(--terra-abstract-modal-horizontal-inset, 10px));
+    outline: none;
+    overflow: hidden;
+    position: fixed;
+    top: 50%;
+    transform: translate(-50%, -50%);
+
+    &.layer-6000 {
+      z-index: 6000 + 1;
```

Comment:
Any reason not to just make this 6001?

Reply:
Not especially, it was the pattern used in modal before, and kind of made a weird kind of sense to have 'layer-6000' be '6000'(+1). I'm fine updating it if people have an opinion either way.